packages/amplify-cli/src/extensions/amplify-helpers/remove-pinpoint-policy.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import { pathManager, JSONUtilities } from 'amplify-cli-core';
+import _ from 'lodash';
+
+const projectBackendDirPath = pathManager.getBackendDirPath();
+const pinpointTemplateFileName = 'pinpoint-cloudformation-template.json';
+
+export function removeGetUserEndpoints(resourceName) {
+  const resourceDirPath = path.join(projectBackendDirPath, 'analytics', resourceName);
+  const pinpointTemplateFilePath = path.join(resourceDirPath, pinpointTemplateFileName);
+  let pinpointTemplateFile: any = JSONUtilities.readJson(pinpointTemplateFilePath);
+  pinpointTemplateFile.Resources.CognitoUnauthPolicy.Properties.PolicyDocument.Statement[0].Action = [
```

Comment:
Loadsh should be used for safe get and set and also remove the entry we don't want not setting all the others.